staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -587,6 +596,48 @@ func PrintFlagsWithDryRunStrategy(printFlags *genericclioptions.PrintFlags, dryR
 	return printFlags
 }
 
+type DeleteStrategy int
```

Comment:
I would just drop this type and use the existing `metav1.DeletePropagation` type since this is a one-to-one mapping.

Reply:
Agree, done.